src/cfml/system/services/ServerService.cfc

```diff
@@ -887,6 +889,13 @@ component accessors="true" singleton {
 			);
 		}
 
+		//ajp enabled with secret
+		if( serverInfo.AJPEnable && len(serverInfo.AJPSecret) ){
```

Comment:
@BalbinoOrtus Very close, but please note this doesn't match our coding rules in regards to whitespace (nor how I typed the code in Slack :) )

Reply:
totally right, and its fixed now